cfme/tests/configure/test_timeprofile.py

```diff
@@ -60,17 +61,11 @@ def test_days_required_error_validation():
         timezone="(GMT-10:00) Hawaii",
         days=False,
         hours=True)
-    if version.current_version() > "5.7":
-        tp.create(cancel=True)
-        assert "At least one day needs to be selected" == \
-               tp.timeprofile_form.days.angular_help_block
-        if version.current_version() > "5.8":
-            assert form_buttons.save.is_dimmed
-        else:
-            assert form_buttons.add.is_dimmed
-    else:
-        with error.expected("At least one Day must be selected"):
-            tp.create()
+    view = navigate_to(st.Timeprofile, 'All')
+    tp.create(cancel=True)
+    if view.timeprofile_form.help_block.read() == "At least one day needs to be selected":
```

Comment:
**REQURIED**
Don't use `read()` method to get the text. This method mostly used by widgetastic internally. Use `text` property instead.